shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -1,379 +1,223 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 "use strict";
-import { objCreateFn, strShimObject, strShimUndefined, strShimFunction, strShimPrototype } from "@microsoft/applicationinsights-shims";
-import { getWindow, getDocument, getCrypto, getPerformance, getMsCrypto, getNavigator }  from './EnvUtils';
+import { objCreateFn, strShimUndefined } from "@microsoft/applicationinsights-shims";
+import { getWindow, getDocument, getPerformance, isIE }  from "./EnvUtils";
+import { 
+    arrForEach, arrIndexOf, arrMap, arrReduce, attachEvent, dateNow, detachEvent, hasOwnProperty, 
+    isArray, isBoolean, isDate, isError, isFunction, isNullOrUndefined, isNumber, isObject, isString, isTypeof, 
+    isUndefined, objDefineAccessors, objKeys, strTrim, toISOString
+} from "./HelperFuncs";
```

Comment:
Splitting these out to their own file to "fix" the cyclic dependencies build issues